src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SourceFeedType.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    internal enum SourceFeedType
+    {
+        Http,
+        LocalAbsolute,
```

Comment:
We already have some FeedType telemetry. 
What makes this more helpful than that?
Can we reuse it?

Reply:
I need local feeds to be more specific (relatvie vs. absolute paths in local feeds). I might reuse it, just didn't because I was afraid of breaking source telemetry.

Reply:
Why does relative vs absolute matter?

Reply:
> good to know

What would we learn from that? 
Would we do anything differently if we're dealing wtih a share vs a local one?

As far as I know, most of our code doesn't care about it. So I can't think why we should care.